app/controllers/users_controller.rb

```diff
@@ -6,6 +6,7 @@ class UsersController < ApplicationController
   def update
     if current_user.update(user_params)
       bypass_sign_in current_user
+      flash[:notice] = I18n.t("devise.passwords.updated")
```

Comment:
Should this fire only on password change rather than on any change (which I think it currently does?)

Reply:
ahh, good point. Let me fix that